ena-submission/src/ena_deposition/create_assembly.py

```diff
@@ -388,12 +390,14 @@ def can_be_revised(config: Config, db_config: SimpleConnectionPool, entry: dict[
             update_assembly_error(db_config, error, seq_key=entry, update_type="revision")
             return False
 
-    differing_fields = []
-    for value in config.manifest_fields_mapping.values():
-        for field in value.get("loculus_fields", []):
-            last_entry = last_version_data[0]["metadata"].get(field)
-            new_entry = entry["metadata"].get(field)
-            if field == "authors":
+    differing_fields = {}
+    for mapping in config.manifest_fields_mapping.values():
+        for loculus_field_names in mapping.get("loculus_fields", []):
```

Comment:
this function is iterating over all fields in the `loculus_fields`, for `authors` it iterates over a list with ['authors']